notebooks/textbook/CHSH_Inequality.ipynb

```diff
@@ -0,0 +1,330 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "fig, ax = plt.subplots(figsize=(8, 5))\n",
+    "plt.plot(angles, chsh_values, \".\", label=\"CHSH Value\")\n",
+    "\n",
+    "plt.grid(which=\"major\", axis=\"both\")\n",
+    "plt.axhline(y=2, color=\"r\", linestyle=\"-\")\n",
+    "#plt.axhline(y=-2, color=\"r\", linestyle=\"-\")\n",
+    "plt.axvline(np.pi / 4, color=\"k\", linestyle=\"-\")\n",
+    "plt.axhline(y=np.sqrt(2) * 2, color=\"k\", linestyle=\"-.\")\n",
+    "#plt.axhline(y=-np.sqrt(2) * 2, color=\"k\", linestyle=\"-.\")\n",
```

Comment:
we can remove these lines since they are commented out 
```suggestion
    "plt.axvline(np.pi / 4, color=\"k\", linestyle=\"-\")\n",
    "plt.axhline(y=np.sqrt(2) * 2, color=\"k\", linestyle=\"-.\")\n",
```